test/sanity/validate-modules/schema.py

```diff
@@ -36,6 +36,24 @@ def sequence_of_sequences(min=None, max=None):
     )
 
 
+seealso_schema = [
+    Any(
+        {
+            Required('module'): str,
+            'description': str,
+        },
+        {
+            Required('ref'): Any(list_string_types),
```

Comment:
I wonder if these should be string (singular) rather than plural `*string_types`

Reply:
Well, there are multiple string types, I don't think we can work around that.